practical-2/skipgram.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, vocab, embedding_dim, use_cuda=False):
         self.use_cuda = use_cuda
 
         if use_cuda:
+            print("sam, fuck you. i made it work with cuda :P.")
```

Comment:
I'm going to link this in the report, I swear :laughing:

Reply:
be my guest :P